core/src/components/modal/test/fixtures.ts

```diff
@@ -16,7 +16,7 @@ export class CardModalPage {
     this.ionModalDidDismiss = await page.spyOnEvent('ionModalDidDismiss');
   }
   async openModalByTrigger(selector: string) {
-    await this.page.click(selector);
+    await this.page.locator(selector).click();
```

Comment:
Why was this change made?

Reply:
I switched it since the Playwright documents [discourage](https://playwright.dev/docs/api/class-page#page-click) the use of `page.click` and suggest `locator.click` instead.

Reply:
Plus it did help lessen the flakiness when switched over. The same for `waitForElementState`.